internal/flightplan/matrix_decoder.go

```diff
@@ -16,7 +16,8 @@ import (
 
 type matrixDecoder struct{}
 
-type DecodedMatrices struct {
+// MatrixBlock represent a full "matrix" block at various stages.
+type MatrixBlock struct {
```

Comment:
I renamed `DecodeMatrices` to `MatrixBlock` and `DecodedScenarioBlock`'(s) to `ScenarioBlock` because they're shorter, more consistent, and clearer.